tests/MSBuildDeviceIntegration/Tests/XASdkDeployTests.cs

```diff
@@ -71,8 +71,6 @@ public class XASdkDeployTests : DeviceTest
 		[TestCaseSource (nameof (DotNetInstallAndRunSource))]
 		public void DotNetInstallAndRun (bool isRelease, bool xamarinForms, string targetFramework)
 		{
-			AssertHasDevices ();
```

Comment:
Some of these if I run them locally and forgot to attach a device, it would give a nice error message. Was there a reason to remove it?

Reply:
I moved this check into DeviceTest setup so we don't need to define it on every test. We will also attempt to acquire an android target during setup now as well if one is not available: https://github.com/xamarin/xamarin-android/pull/7731/files/0ffe290ec63c4eb15ed1bb7b44811d7ba6007125#diff-b90ec706ed5e04747e6db2983d908cf1dcc5e7ca4fae94575961905975dc886fR97